### PR TITLE
EUI-3616 - Mock sorting

### PR DESCRIPTION
**Pull request by connorpgpmcelroy**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-3616
https://tools.hmcts.net/jira/browse/EUI-3896
https://tools.hmcts.net/jira/browse/EUI-3897
https://tools.hmcts.net/jira/browse/EUI-3851

### Change description ###
Enable sorting in release 2 - Existing functionality used - Work composed only of mocking data
Reset sorting + persist sorting between pages

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x ] No
```

### CODE REVIEW

File: src/work-allocation-2/containers/task-list/task-list.component.html [hunk 1]
```diff
@@ -13,6 +13,9 @@
     <span class="text-16" id="search-result-summary__text">Displaying <span class="govuk-!-font-weight-bold">{{ tasksTotal || 0 }} </span>tasks.</span></div>
 </ng-template>
 
+<!-- Reset Sorting Button -->
+<div *ngIf="showResetSortButton" style="float: right; margin-bottom: -10px; margin-top: -10px;"><button type="button" class="govuk-button hmcts-button--secondary" (click)="onResetSorting()">Reset sorting</button></div>
```
Comment (rayliangatos) [inline src/work-allocation-2/containers/task-list/task-list.component.html:17]: could you move the style to scss?

Comment (udaydenduluri33) [reply src/work-allocation-2/containers/task-list/task-list.component.html:17]: yeah agree with @rayliangatos

Comment (connorpgpmcelroy) [reply src/work-allocation-2/containers/task-list/task-list.component.html]: Done

**Review by rayliangatos — APPROVED** [review]

**Review by udaydenduluri33 — APPROVED** [review]
LGTMc&l

**Pull request merged**